requirements/constraints.txt

```diff
@@ -121,7 +121,7 @@ libsass==0.10.0
 click==8.1.6
 
 # pinning this version to avoid updates while the library is being developed
-openedx-learning==0.2.3
+openedx-learning==0.2.4
```

Comment:
I'm not sure why it's showing this in the diff, because `master` is already updated to use `0.2.4`

Reply:
I updated this branch with `master`